src/platform/Zephyr/GenericBLEManagerImpl_Zephyr.ipp

```diff
@@ -144,6 +144,11 @@ void GenericBLEManagerImpl_Zephyr<ImplClass>::DriveBLEState()
             ClearFlag(mFlags, kFlag_AdvertisingEnabled);
             ChipLogProgress(DeviceLayer, "CHIPoBLE advertising disabled because device is fully provisioned");
         }
+        if(ConnectivityMgr().IsThreadProvisioned())
```

Comment:
Seems that both if statements result in the same code, right? Can we combine them to just one if-statement?